tensorflow/python/data/experimental/ops/optimization_options.py

```diff
@@ -188,11 +188,9 @@ class OptimizationOptions(options.OptionsBase):
       "cardinality preserved transformations, e.g. dataset.map(...).take(3)"
       "will be optimized to dataset.take(3).map(...). For now this"
       "optimization will move `skip`, `shard` and `take` to the front of"
-      "`cache`, `map` and `prefetch`. And notice this optimization is only"
-      "for performance, it will not affect the output of the dataset."
-      "However, it will influence the cache to the file, for the unused"
-      "data will no longer be saved after this optimization."
-      "If None, defaults to False.")
+      "`map` and `prefetch`. And notice this optimization is only for"
+      "performance, it will not affect the output of the dataset."
```

Comment:
each line needs a space at the end so that the last word of the line doesn't get combined with the first word of the next line.